libraries/gtk-sharp3/gtk-sharp3.SlackBuild

```diff
@@ -80,26 +80,36 @@ find -L . \
  \( -perm 666 -o -perm 664 -o -perm 640 -o -perm 600 -o -perm 444 \
   -o -perm 440 -o -perm 400 \) -exec chmod 644 {} \+
 
-# Switch away from deprecated gmcs mono compiler
-sed -i 's|gmcs|mcs|' configure configure.ac
-
-# Fix compile issues
-sed -i 's|MONO_PROFILE_ENTER_LEAVE|1 << 12|' gtk/gui-thread-check/profiler/gui-thread-check.c
-sed -i '12i \\nusing Range=Gtk.Range;' sample/test/TestRange.cs
-
-CFLAGS="$SLKCFLAGS" \
-CXXFLAGS="$SLKCFLAGS" \
-./configure \
-  --prefix=/usr \
-  --libdir=/usr/lib${LIBDIRSUFFIX} \
-  --sysconfdir=/etc \
-  --localstatedir=/var \
-  --docdir=/usr/doc/$PRGNAM-$VERSION \
-  --disable-static \
-  --build=$ARCH-slackware-linux
-
-make
-make install DESTDIR=$PKG
+# Correct outdated references
+sed -i 's|gdk_api_includes|gio_api_includes|' \
+  Source/{gdk,gio,gtk}/generated/meson.build \
+  Source/sample/valtest/generated/meson.build
+
+# Mono packages use /usr/lib regardless of arch
+patch -p 1 < $CWD/gtk-sharp3-nolibdir.patch
```

Comment:
@bassmadrigal Looks like you missed staging this patch into git.

Reply:
Just added. Thanks for the catch!